packages/history-service/src/create-root-location-transformer.ts

```diff
@@ -128,6 +137,29 @@ export function createRootLocationTransformer(
       return getConsumerPath(consumerPaths, consumerUid);
     },
 
```

Comment:
I won't go further into stylistic problems, as this code has obviously not been TSLint checked and Prettier formatted. Please configure your IDE accordingly or use the available commands from package.json.

Reply:
done